src/views/pacbio/PacbioReceptionSamplesExtraction.vue

```diff
@@ -81,19 +82,29 @@ export default {
       this.busy = true
       try {
         let getSETubeResponse = await this.getSampleExtractionTubesForBarcodes(this.barcodeArray)
-        if (!getSETubeResponse.successful || getSETubeResponse.empty) {
-          throw getSETubeResponse.errors
+
+        if (!getSETubeResponse.success || getSETubeResponse.data.data.length === 0) {
```

Comment:
Based on the associated changes in the action this could now be:

```
if (!getSETubeResponse.success) {
 throw { message: getSETubeResponse.errors }
}
```